doc/features.mld

```diff
@@ -106,13 +106,13 @@ end
 module type B = A
 ]}
 
-For these simple module type declarations, where the right-hand side is just a
+For these simple module-type declarations, where the right-hand side is just a
 path, [odoc] treats them as module aliases and doesn’t produce an expansion.
 This example is rendered {{!Odoc_examples.Expansion.ModuleTypeAliases}here}.
 
 When strengthening, OCaml turns modules into aliases to the original
 module, but nothing is done to module types. In contrast, [odoc] replaces
```

Comment:
```suggestion
When strengthening, OCaml turns modules into aliases of the original
module, but nothing is done to module types. In contrast, [odoc] replaces
```
Should this work be "of" instead of "to"? 
Is OCaml make an alias of the original module, perhaps "OCaml creates alias modules from the original module," or "OCaml takes the original module and turns it into an alias."

Let's clarify this a little more. I suspect the same will go for the next in li 114-115
"In contrast, `odoc` replaces module types with 'aliases' of the originals," rather than "to" the originals.